### PR TITLE
W: [fr] Linux Install Script

### PR DESCRIPTION
**Pull request by gegeweb**
@trebmuh ready for review

### CODE REVIEW

File: wiki/fr/fr-Linux-Install-Script.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+# set DISTRO either to "Ubuntu", "Debian" or "Fedora"
+DISTRO="Debian"
+LINVERSION="10.6"
+```
+
+Après, testez l'installation sous debian et modifiez les commandes pour que le script d'installation fonctionne sous Debian. Partagez, s'il vous plait, vos scipts d'installation fonctionnels sur ce Wiki. Le responsable du dépot peut y ajouter un répertoire `/install_scripts` pour les scripts d'installation fonctionnels. Créez une demande de fusion pour votre nouveau script d'installation ou ouvrez une « issue » pour demander l'ajout du script documenté et testé dans ce dépôt. 
+
+### Le script d'installation
+
+Copiez le script d'installation suivant dans un ficher et suvegardez le avec le nom `install4ubuntu18_4.sh`. Après avoir sauvegardé ce fichier dans, par ex., votre répertoire `Téléchargements`, placez vous dans ce répertoire et appelez le script avec `sh install4ubuntu18_4.sh`.
```
Comment (trebmuh) [inline wiki/fr/fr-Linux-Install-Script.md:50]: * suvegardez -> sauvegardez
* "appelez le script avec " -> "appelez le script suivant avec "

File: wiki/fr/fr-Linux-Install-Script.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+fi
+
+```  
+
+## Améliorations possibles du script d'installation pour Jamulus
+
+Le script peut, au tout début, demander pour quelle distribution Linux il doit être appelé.  
+Ça peut être testé avec la commande `lsb_release`.
+
+Avec la commande `lsb_release` qui retourne les informations spécifiques à la ditribution sous Linux.
```
Comment (trebmuh) [inline wiki/fr/fr-Linux-Install-Script.md:137]: * "ditribution" -> "distribution"

File: wiki/fr/fr-Linux-Install-Script.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+```bash
+$ lsb_release -a
+No LSB modules are available.
+Distributor ID: Ubuntu
+Description:    Ubuntu 11.04
+Release:        11.04
+Codename:       natty
+```
+
+La problématique est que la commande `lsb_release` doit être disponible sur le système Linux. Pour les systèmes basés sur CentOS ou Fedora la commande `lsb_release` n'est disponible que si les paquets de base `lsb` sont installés. La détéction automatique de la version de Linux pourrait donc ne pas fonctionner.
```
Comment (trebmuh) [inline wiki/fr/fr-Linux-Install-Script.md:150]: * "détéctioné -> détection"

File: wiki/fr/fr-Linux-Install-Script.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+En conséquece, obtenir les valeurs de `DISTRO` et `LINVERION` avec la commande `read` devrait être la meilleure amélioration dépendante de la distribution plutôt que d'automatiser la configuration avec `lsb_release`.
```
Comment (trebmuh) [inline wiki/fr/fr-Linux-Install-Script.md:152]: * "conséquece" -> "conséquence"
* "devrait être" -> "serait"
* dépendante -> "en fonction"

**Review by trebmuh — CHANGES_REQUESTED** [review]
Thanks for the initial work.
Good that it's sticked close to the EN text.

Please note that there are a bunch of typos which could be avoided with using an automatic corrector.

**Comment by gegeweb** [comment]
> Please note that there are a bunch of typos which could be avoided with using an automatic corrector.

If you know one working in VSCode…

Everithing should be fixed.

File: wiki/fr/fr-Linux-Install-Script.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+layout: wiki
+title: "Trucs et astuces Jamulus – Script bash d'installation pour Linux"
+lang: "fr"
+permalink: "/wiki/Linux-Client-Install-Script"
+---
+
+
+# Script bash pour télécharger et compiler sous Linux
+
+Si vous prévoyez d'installer Jamulus sur plusieurs machnine Linux, vous pourrier avoir envie d'éssayer ce script.
```
[the remark this answers is not in the document]
Comment (trebmuh) [reply wiki/fr/fr-Linux-Install-Script.md]: There is still a typo here: mach**n**ines.

File: wiki/fr/fr-Linux-Install-Script.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+L'exemple suivant a été testé sous Linux Mint et combine toutes les commandes ci-dessous en un seul script pour Ubuntu/Linux. Pour pouvoir incorporer les commandes spécifiques aux différents distributions Linux des variables définissent la distribution et la version pour lesquelles ce script d'installation pourrait être utilisé. L'exemple utilise Ubuntu version 18.04 comme référence. Donc le nom suggéré pour le script est, par ex., `install4ubuntu18_4.sh`. Les commandes du script sont génériques afin que le script puisse être modifié pour fonctionner avec d'autres distributions.
```
[the remark this answers is not in the document]
Comment (trebmuh) [reply wiki/fr/fr-Linux-Install-Script.md]: Donc -> Ainsi

File: wiki/fr/fr-Linux-Install-Script.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+if [ "$LINVERSION"  = "18.4" ]
+then
+    echo "Perform Installation Specifics for $DISTRO Version $DISTRO" 
+fi  
+```
+
+La variable `LINVERSION` n'est actuellement pas utilisé dans le script suivant, c'est juste une démonstration sur la manière dont les commandes spécifiques à une version peuvent être utilisées.
+
+### Adaptation du script d'installation
+Si vous voulez créer un script d'installation pour debian, copiez simplement le script `install4ubuntu18_4.sh` en le renommant `install4debian10_6.sh` et modifez la varable distribution :
```
[the remark this answers is not in the document]
Comment (trebmuh) [reply wiki/fr/fr-Linux-Install-Script.md]: Seems you forget those 2 ones.

[the remark this answers is not in the document]
Comment (gegeweb) [reply wiki/fr/fr-Linux-Install-Script.md]: Should be good now.

**Comment by trebmuh** [comment]
Almost. Still a few typo here and there.

> If you know one working in VSCode…

I don't use this. Maybe you can:
1. make the translation in whatever software suiting your needs
2. load the file in a text editor with a spelling checker

Even firefox (I'm using the github web interface to do review) underlines typos.

**Comment by trebmuh** [comment]
You've been quick and resolved those even before my last message! Thanks.

@ann0see : greenlight for merging this one.

**Comment by gegeweb** [comment]
> Even firefox (I'm using the github web interface to do review) underlines typos.

Thanks for the idea.

**Comment by gegeweb** [comment]
@ann0see green light for me too.

**Review by ann0see — APPROVED** [review]
BTW: this could also go to the new community section.

**Pull request merged**